src/app/settings-components/install/install.component.ts

```diff
@@ -67,21 +68,22 @@ export class InstallComponent implements OnDestroy {
   moveToGameVersionTab(needsInstall: boolean) {
     if (needsInstall)
       this.needsIsoInstall = needsInstall;
-    this.tabForceSet = true;
-    this.tab = 1;
-
+    // this.tabForceSet = true;
+    // this.tab = 1;
```

Comment:
Why was this commented out for the PR?

Reply:
Because it would keep switching the tab every time the verify path button was pressed and the path determined to be wrong. I assume it was there to prompt users to download a game version if they dont have one installed in the current directory, but it was just annoying.

I don't know if it's necessary, I didnt think so, but it could be added back. However the verify path button would need a rework.

Reply:
To be honest, this button isn't really ready to go in a release yet, because it doesn't update the game version when you change & verify the install path. So even if you pick a completely stupid path, it will still say `Game: v0.2.0` or whatever, maybe until you relaunch the launcher. The button does what it was intended to do, but it should probably do that as well.

Reply:
Looking into it, I can see that commenting it out was probably stupid, since it is used more than just when pressing the verify path button. I will uncomment it and revisit the button again.